### PR TITLE
[orchagent] Add zmq sync mode flag

### PR DESCRIPTION
**Pull request by kcudnik**
**What I did**
Added support for zmq synchronous mode flag, passed from command line to sairedis

**Why I did it**
To add easy access for zmq sync mode from command line

**How I verified it**
Run OA with -z flag and syncd with -z flag

**Details if related**

### CODE REVIEW

**Comment by kcudnik** [comment]
requires https://github.com/Azure/sonic-sairedis/pull/711 to wrok

**Comment by kcudnik** [comment]
retest vs please

**Comment by qiluo-msft** [comment]
@shi-su Could you help review?

File: orchagent/main.cpp [hunk 1]
```diff
@@ -76,6 +77,7 @@ void usage()
     cout << "    -m MAC: set switch MAC address" << endl;
     cout << "    -i INST_ID: set the ASIC instance_id in multi-asic platform" << endl;
     cout << "    -s: enable synchronous mode" << endl;
```
Comment (shi-su) [inline orchagent/main.cpp:79]: Could you also add a message that this is being deprecated as well?

File: orchagent/main.cpp [hunk 2]
```diff
@@ -76,6 +77,7 @@ void usage()
...
+    cout << "    -z: redis communication mode (redis_async|redys_sync|zmq_sync), default: redis_async" << endl;
```
Comment (shi-su) [inline orchagent/main.cpp:80]: Seems "redys_sync" is a typo.

Comment (kcudnik) [reply orchagent/main.cpp]: done

File: orchagent/main.cpp (hunk 1 above)
Comment (kcudnik) [reply orchagent/main.cpp]: done

**Comment by kcudnik** [comment]
retest vs please

**Comment by kcudnik** [comment]
retest vs please

**Review by shi-su — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**